### DISCUSSION TITLE
Error in compiling source code with gfortran compiler

### DISCUSSION THREAD

**Discussion by jczhao1022** [OpenRadioss/OpenRadioss#1745] [Q&A] [answered]
Error in compiling starter: array subscript 0 is outside array bounds
![image](https://github.com/OpenRadioss/OpenRadioss/assets/120549735/baf9ed7c-8571-4eb0-9083-b801fc423d90)

**Comment by jczhao1022** [comment, marked as answer]
Up to g++-11 gcc-11 gfortran-11,  there was no warning in above code.

**Reply by laurent-altr** [reply]
Thank you, 
Here is the explanation: 
Recently we added few compiler flags, but these flags does not work for older/unsupported version of GCC.
